nb2workflow/nbadapter.py

```diff
@@ -407,12 +407,13 @@ def update_summary(self, **d):
 
         
 
-    def execute(self, parameters, progress_bar = True, log_output = True, inplace=False):
+    def execute(self, parameters, progress_bar = True, log_output = True, inplace=False, callback=None):
```

Comment:
Parameter name `callback` is a bit misleading, `callback_url` is probably better
The same applies to `_pass_callback()` method.

Reply:
I guess also for the `_execute` method?